apps/picsa-apps/dashboard/src/app/modules/crop-information/pages/new_entry/new_entry.page.ts

```diff
@@ -0,0 +1,55 @@
+import { CommonModule } from '@angular/common';
+import { Component, OnInit } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { Router } from '@angular/router';
+
+import { DashboardMaterialModule } from '../../../../material.module';
+import { CropProbabilityDashboardService } from '../../crop-information.service';
+
+@Component({
+  selector: 'dashboard-new-entry',
+  standalone: true,
+  imports: [CommonModule, DashboardMaterialModule, RouterModule, FormsModule, ReactiveFormsModule],
+  templateUrl: './new_entry.component.html',
+  styleUrls: ['./new_entry.component.scss'],
+})
+export class NewEntryPageComponent implements OnInit {
+  station_name: string;
+  water_lower: number;
+  water_upper: number;
+  length_lower: number;
+  length_upper: number;
+  station_notes: string[];
```

Comment:
nit(blocking)
It would be really nice if we could manage the data bindings as a single `form` object that can also be used to handle things like validation. 

Could you try implement as a reactive form instead, there should be some good guidance to create the form using the angular formbuilder at: https://angular.dev/guide/forms/reactive-forms#using-the-formbuilder-service-to-generate-controls

Example of using the generated form with ngModel inputs: 
https://angular.dev/guide/forms/reactive-forms#associate-the-formgroup-model-and-view